src/segment/stories/Segment.stories.tsx

```diff
@@ -1,40 +1,49 @@
+import "./Segment.css";
 import * as React from "react";
-import { Meta } from "@storybook/react";
+import { Meta, Story } from "@storybook/react";
+import { DEFAULT_REACT_CODESANDBOX } from "storybook-addon-preview";
 
-import "./index.css";
-import { Segment } from "../Segment";
-import { SegmentField } from "../SegmentField";
-import { useSegmentState, SegmentStateProps } from "../SegmentState";
+import { App as Segment } from "./Segment.component";
+import { appTemplate, appTemplateJs, cssTemplate } from "./templates";
 
 export default {
+  component: Segment,
   title: "Segment",
+  parameters: {
+    preview: [
+      {
+        tab: "ReactJS",
+        template: appTemplateJs,
+        language: "jsx",
+        copy: true,
+        codesandbox: DEFAULT_REACT_CODESANDBOX(["renderless-components@alpha"]),
+      },
+      {
+        tab: "React",
+        template: appTemplate,
+        language: "tsx",
+        copy: true,
+        codesandbox: DEFAULT_REACT_CODESANDBOX(["renderless-components@alpha"]),
+      },
+      {
```

Comment:
we will use the createPreviewTabs helper after merging #136

Reply:
👍